docs: Rename Best Practiced to Guides; Add Try Out with Gitpod (#275)
askui/askui: docs/docs/general/08-Terminology.md

| Term                   | Meaning     |
| ---------------------- | ----------- |
| annotation             | Marked area around an element with metadata name, text, and bounding box coordinates. |
| access token           | Gives you access to the askui inference in combination with your `workspace id`. Every `access token` has an expiry date and is assigned to exactly one workspace.  |
| automation             | A set of instructions to achieve a task. |
| bounding box           | Rectangle described by coordinates that describe an element's location. Displayed visually as red rectangle. |
| command                | An action on the user interface. For example mouse left click or keypress. |
| credentials            | Combination of `workspace id` and `access token` living in the same workspace. |

> | element                | Or user interface element. Building blocks websites or apps are build from. |

Reviewer: ```suggestion
| element                |  A user interface component which websites and blocks are build from. |
```

Author: Much simpler! Great.

> | filters                | Type of an element. For example _button_. |

Reviewer: ```suggestion
| filters                | Screen for a type of an element. For example _button_. |
```

Author: I see why you made the change but I do not understand what **Screen** means here 😄 . @thinh-askui would you explain, please?

Reviewer: Sorry, should be *screens. First I wanted to write "Filters for a type of an element."
Idea was just to use a synonym for filter. Maybe it's not the best synonym :D

Author: We can go with **screens**.
